src/events/events.js

```diff
@@ -0,0 +1,5 @@
+
+export const showList = () => ((
+  document.querySelector('.goodsList').classList.remove('disable'),
+  document.querySelector('.goodsButton').classList.add('disable')
```

Comment:
Don't use `querySelector` in the React components.
You could use state and conditional rendering for that.

Reply:
![image](https://user-images.githubusercontent.com/63505873/86327490-1ab7a200-bc4c-11ea-88ce-7d18c0a5097c.png)